handbook/product/roadmap.md

```diff
@@ -82,14 +89,12 @@ TO DO
 
 ## Web
 
-TO DO: below are rough items from a previous web team planning discussion.
-
-1. Webapp consistency
-1. The extension registry is confusing and does not communicate it's value
-1. Many developers do not realize Sourcegraph has a browser extension
-1. The Sourcegraph extension development experience is confusing and there aren't any good examples
-1. A few high quality non-language intelligence extension would help communicate the value and opportunity of Sourcegraph extensions
-1. Code insights
-1. Add support for more code hosts with the browser extension (e.g., Gerrit, Bitbucket Cloud).
-1. Provide search capabilities from the code host
-1. IDE integrations
+1. ✅ Existing sourcegraph extensions are more discoverable ([RFC 209](https://docs.google.com/document/d/1I5BMEGp3QuB81AjSzLCQwq_XJV1sXevlU0lpB4O1pj8/edit#))
+1. ✅ The Sourcegraph browser extension is more discoverable and easy to congifure ([RFC 221](https://docs.google.com/document/d/19f4xleYBU1zZZdqMmXlLmFxeR-fwEpOwTOgViOFOnyo/edit))
+1. 🔄 Build new and improved Sourcegraph extensions to showcase the value and opportunity of extensions ([RFC 246](https://docs.google.com/document/d/1HngEeLNAe7_QzVJr6UPi0Si4ZALqTzb7uonOxUiJP6g/edit))
+1. Improve the Sourcegraph extensions (internal) development experience ([RFC 155](https://docs.google.com/document/d/1ikrUNVe3YVbR-JpegxhjrFdmRkTGzTLcOMkKHnOyjuE/edit)) and (external) documentation
+1. Code insights prototyping (TBD)
```

Comment:
Can we remove the "prototyping" qualifier here? A prototype is what we have right now, as expressed I think what we should be working on is to reach something (an API) that we could give customers to play with outside a curated demo environment, like an alpha, which allows us to be more iterative and get feedback. I don't think it's valuable to invest more into prototyping at this point in time (as in something we can use in demos) as I don't see what questions that would help us answer.
```suggestion
1. Code insights (TBD)
```

Reply:
I'll remove it since I'm not particularly married to this word, but to me "prototyping" is not just "building a prototype" in code but also can encompass low-fi or high-fi mocks, customer interview exercises/interactive simulations, and basically anything beyond a verbal/written q&a that helps us answer questions.

Reply:
I would also consider that an alpha API is essentially a "prototype" API, but I'll need another week or so before I'm set on what questions I want answered next, and don't want to commit to an API as the next build by inputting "API" here. (Even if it is obvious to you that API is what we need next, since you've had so much more time to think/hear about insights, in which case, awesome – I'm sure it will then become obvious to me. Definitely seeing lots of conversations around an API and think it's certainly part of code insights, I just don't know if it's the very first part of code insights.)